addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -92,6 +92,16 @@ def action_quit(self):
         self.ensure_one()
         return self.write({'user_ids': [(3, self._uid)]})
 
+    def action_view_channel_ratings(self):
+        self.ensure_one()
+        action = self.env['ir.actions.act_window']._for_xml_id('im_livechat.rating_rating_action_view_livechat_rating')
+        action['display_name'] = _('Ratings for %s') % self.name
+        action['domain'] = [('parent_res_id', '=', self.id), ('parent_res_model', '=', self._name), ('consumed', '=', True)]
```

Comment:
`self._name` is not dynamic, it's always `im_livechat.channel` so I would use it explicitly for clarity.

Reply:
But ideally this should just take the original domain (if you don't remove it) and append the filter on `parent_res_id`.